tsconfig.json

```
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "checkJs": true,
+    "esModuleInterop": true,
+    "moduleResolution": "node",
+    "noImplicitAny": false,
```

```suggestion
    "noImplicitAny": true,
```

This is one of the most important checks of TS. As we discussed yesterday, if you have implicit anys, your code is fundamentally unsafe.